heroku-24/setup.sh

```diff
@@ -119,6 +123,12 @@ packages=(
   zstd
 )
 
+if [[ $(dpkg --print-architecture) == "amd64" ]]; then
+  packages+=(
+    libgirepository-1.0-1 # Because libgirepository1.0-dev and libgirepository-1.0-dev are in the build image
+  )
+fi
```

Comment:
Prior to this architecture-conditional addition, the ARM64 and AMD64 **run** images contained exactly the same packages (with the exception of the binutils package name including the arch so being named `binutils-x86-64-linux-gnu` vs `binutils-aarch64-linux-gnu` - but effectively the package provides the same functionality). Now their contents will vary by arch. 

Whilst I know that (a) the run images containing the same packages isn't something that we can necessarily guarantee for the future (eg Heroku-26), and (b) the **build** images already vary by arch (in part due to `libgirepository-1.0-dev`), but there is still some appeal/simplicity to me of the packages list being the same - if only to make things simpler when we display the table of packages on Dev Center (and to avoid user confusion when they try to work out what's available for their app, without them having to know what arch they are using).

Given that `libgirepository-1.0-1` is only needed because of `libgirepository-1.0-dev`, and the headers are only pulled in because of slightly obscure reasons (it's a transitive dependency of `libmagickwand-dev` and friends on one arch only), I wonder if we can just skip installing this package? I don't think it's the type of package anything will depend on at runtime. (Plus it's not going to be available on ARM64 anyway)

Reply:
Yeah I prioritized consistency between run and build image over consistency across architectures here, since the build images already differ, so we have the problem for the Dev Center packages list already.

I dug through the various Debian control files for a bit to see if the `libgirepository-1.0-dev` dependency difference between architectures is even intentional, but couldn't find something conclusive in a reasonable amount of time.

Happy to drop this one instead.